src/components/Sidebar/contentTree.js

```diff
@@ -82,7 +87,7 @@ const calculateNavigationFluentUI = (calculatedNavigation, pathname) => {
       sourceNavGroup.id !== extractLevelGroupID(pathname)[0];
     
     const navLinkGroup = {
-      name: emoji.emojify(sourceNavGroup.title),
+      name: `${emoji.emojify(sourceNavGroup.title)}`,
```

Comment:
I don't think we need string interpolation here, please revert to old statement.
```js
      name: emoji.emojify(sourceNavGroup.title),
```

Reply:
Oh, Yes